html/themes/custom/common_design_subtheme/components/rw-moderation/rw-moderation.css

```diff
@@ -0,0 +1,246 @@
+/**
+ * Moderation backend form.
+ *
+ * Style the moderation forms used by the editorial teams to review posted
+ * documents.
+ */
+/* Hide visually the filters title to reduce scrolling. */
+.path-moderation .rw-moderation-content-filters {
+  padding: 24px 0 0 0;
+}
+.path-moderation .rw-moderation-filters > legend {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  white-space: nowrap;
+  border: none;
+  clip-path: inset(100%);
+}
+.path-moderation #edit-filters {
+  padding-bottom: 0;
+  border-bottom: none;
+}
+.path-moderation #edit-filters-status [class*="form-item-filters-status-"] {
+  width: 100%;
+  margin-bottom: 12px;
+  padding: 2px 8px;
+  white-space: nowrap;
+  text-transform: none;
+  border-left-width: 8px;
+  background: none;
+  font-size: 16px;
+  line-height: 16px;
+}
+.path-moderation #edit-filters-status.form-checkboxes {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(140px, 1fr));
+  grid-gap: 0 24px;
+  margin-bottom: -12px;
+}
+.path-moderation.path-moderation-content-disaster #edit-filters-status .form-checkboxes {
+  grid-template-columns: repeat(auto-fill, minmax(170px, 1fr));
+}
+.path-moderation #edit-filters-other .fieldset-wrapper > * {
+  display: inline-block;
+  margin-top: 0;
+  vertical-align: middle;
+}
+.path-moderation #edit-omnibox .fieldset-wrapper {
+  display: table;
+}
+.path-moderation #edit-omnibox .fieldset-wrapper > * {
+  display: table-cell;
+  height: 32px;
+  margin: 0;
+  padding: 0;
+  vertical-align: middle;
+}
+.path-moderation #edit-omnibox .fieldset-wrapper > :first-child {
+  padding-right: 8px;
+}
+.path-moderation #edit-omnibox .fieldset-wrapper select {
+  width: auto;
+  height: 100%;
+}
+.path-moderation #edit-omnibox div[data-autocomplete] > input {
+  border-right: 2px solid #e6ecef;
+}
+.path-moderation [data-selection] > span span + span {
+  font-weight: bold;
+}
+.path-moderation .form-actions {
+  margin-top: 16px;
+}
+/* Link to create a new entity. */
+.path-moderation .form-actions a {
+  display: inline-block;
+  margin: 8px;
+  padding: 4px 8px;
+  text-transform: lowercase;
+  border: 2px solid #055372;
+  border-radius: 8px;
+  line-height: 20px;
+}
+.path-moderation form #edit-actions {
+  padding: 12px;
+  text-align: center;
+  border: 1px solid #e6ecef;
+  background: #e6ecef;
+  background: rgba(230, 236, 239, 0.4);
+}
+
+/**
+ * Double datepicker widget.
```

Comment:
@left23 for info, this can be removed I think as the rules for the dual datepicker were already ported to the `rw-datepicker` component: https://github.com/UN-OCHA/rwint9-site/blob/54f75af03b3c661a0b6c4438d55017daa92a0d06/html/themes/custom/common_design_subtheme/components/rw-datepicker/rw-datepicker.css#L159:

<img width="576" alt="Screen Shot 2021-10-12 at 11 21 02" src="https://user-images.githubusercontent.com/696348/136880362-8d36de26-f18e-4c79-be6f-dc5455c97129.png">